README.md

```diff
@@ -0,0 +1,214 @@
+<a name="readme-top"></a>
+
+  <h3><b>Capstone-project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [capstone-project] <a name="about-project"></a>
+
+**[WEBPACK Project]**
+  This capstone  is a project created using html, css and javascript modules with Es6 and  with Webpack and jest installed in it for the better performance while in development even in production mode, because all functionality are imprinted   in here. This capstone is an Advanced one that Working with Api  for file enter changeable and best practice in using ES6.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">HTML5</a></li>
+    <li><a href="">CSS3</a></li>
+    <li><a href="">Modern JavaScript ES6 modules</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">n/a</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="/">n/a</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[initialize folder with npm ]**
+- **[working with webpack]**
+- **[add dev-server using webpack to watch changes ]**
+- **[add webpack.config.js that helps to make some configuration to the project]**
+- **[add css-loader package]**
+- **[Add html, css,javascript to the the project ]**
+- **[add Score using Api ]**
+
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+ <p><a href="https://drive.google.com/file/d/14zRoHM5TOujrdnlNwP1o-A8dBot-5ylN/view?usp=sharing">video description</a></p>
```

Comment:
- [ ] [OPTIONAL] The sound quality of your recorded video was not that good, it can help if the one who is not talking mutes the speaker to avoid sound feedback.